src/compo/pace_aod2ioda.py

```diff
@@ -313,7 +351,7 @@ def main():
         type=str, required=True)
     required.add_argument(
         '--retrieval_method',
-        help="name of retrieval method: uaa, remotap",
+        help="name of retrieval method: uaa, remotap, fmapol",
```

Comment:
```suggestion
        help="name of retrieval method: uaa, remotap, fmapol",
        choices=['uaa', 'remotap', 'fmapol'],
```

this will require one of these 3 arguments specifically... can this be autodetected in the files themselves?